roles/ci_nmstate/molecule/default/converge.yml

```diff
@@ -29,7 +31,12 @@
         name: crc
         ansible_ssh_private_key_file: "{{ ansible_user_dir }}/.crc/machines/crc/id_ecdsa"
         ansible_ssh_user: core
+        cifmw_molecule_ci_nmstate_crc_mac: "{{ cifmw_molecule_ci_nmstate_crc_mac }}"
 
+    - name: Add a dummy ethernet interface to target the crc NMstate
```

Comment:
I'd do that while CRC is still stopped - iirc we start it from within the "prepare.yml" ?
There might be some issues with the amount of available PCI ports or something, with "hot-plugging" the NIC.

Reply:
OK with comment to warn about the PCI thingy.